//catching the error in this class
class Failure {
  final String message;

  Failure([this.message = "An unexpected error occurred"]);
}
